src/lib/store/PropsStore.tests.ts

```diff
@@ -0,0 +1,59 @@
+import { propsStore } from "./PropsStore"
+
+describe("propsStore", () => {
+  describe(propsStore.setPendingProps, () => {
+    it("sets pendingProps", () => {
```

Comment:
nitpick: would be nice to be a little more descriptive for people who come to this module and aren't as familiar
```suggestion
    it("stores and retrieves props for a module", () => {
```